common.go

```diff
@@ -70,3 +90,175 @@ func GatherMetadata(wh *workloads.WorkloadHelper, alerting bool) error {
 	}
 	return nil
 }
+
+// GenerateSSHKeyPair generates an SSH key pair and saves them to the specified files
+func generateSSHKeyPair(sshKeyPairPath string) (string, string, error) {
```

Comment:
Can this go into `go-commons` please? Looks like nice function to be reused across our tooling.